config/defaults.conf

```diff
@@ -1,50 +1,56 @@
 // Default config file.
```

Comment:
Pretty much all of these changes are documentation and ordering, though there are a few slightly substantial changes, such as turning off ELMo by default.